<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Error Handling: try and catch</title>
  </head>
  <body>
    <script>
      setTimeout(() => {
        console.log("Hacking wifi please wait");
      }, 2000);
      //Try catch is only run with syncrounus
      try {
        setTimeout(() => {
          try {
            console.log(hassan);
          } catch (error) {
            console.log(error);
          }
        }, 1000);
      } catch (error) {
        console.log(error);
      }
      setTimeout(() => {
        console.log("Hacking azan phone ....");
      }, 3000);
      setTimeout(() => {
        console.log("azan phone is ......");
      }, 4000);
      setTimeout(() => {
        console.log("fetched completed the number is 3475934");
      }, 5000);
    </script>
  </body>
</html>
